### PR TITLE
Fix from @DokuKaefer

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

**Comment by fabianfrz** [comment]
@fichtner Are there any issues?

**Comment by fichtner** [comment]
yes, not a lot of time at the moment

File: src/www/vpn_openvpn_server.php [hunk 1]
```diff
@@ -1508,19 +1508,19 @@ function tuntap_change() {
                         <input name="dns_server_enable" type="checkbox" id="dns_server_enable" value="yes" <?=!empty($pconfig['dns_server1']) || !empty($pconfig['dns_server2']) || !empty($pconfig['dns_server3']) || !empty($pconfig['dns_server4']) ? "checked=\"checked\"" : "" ;?> onclick="dns_server_change()" />
                         <div id="dns_server_data" summary="dns servers">
                               <span>
-                                <?=gettext("Server"); ?> #1:&nbsp;
+                                <?=gettext("Server #1:"); ?>&nbsp;
```
Comment (fichtner) [inline src/www/vpn_openvpn_server.php]: In the future these things should be set programatically: sprintf(gettext('Server #%s'), 1); etc. (will merge now anyway, it's only 4 strings).

File: src/www/vpn_ipsec_phase1.php [hunk 2]
```diff
@@ -716,7 +716,7 @@ function dpdchkbox_change() {
                       <input name="descr" type="text" id="descr" size="40" value="<?=$pconfig['descr'];?>" />
                       <div class="hidden" for="help_for_remotegw">
                         <?=gettext("You may enter a description here " .
-                                                "for your reference (not parsed)"); ?>.
+                                                "for your reference (not parsed)."); ?>
```
Comment (fichtner) [inline src/www/vpn_ipsec_phase1.php]: Merge lines where you can, improves readability and reduces potential errors.

**Comment by fichtner** [comment]
Purrfect, thanks!

**Pull request merged**